src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -403,24 +403,27 @@ public StudyEntity doGetStudyWithPreFetchedLoadFlowResultAndUpdateIsPrivate(UUID
                 .log(ROOT_CATEGORY_REACTOR, Level.FINE);
     }
 
-    private Mono<? extends Throwable> handleStudyCreationError(UUID studyUuid, String studyName, String userId, boolean isPrivate, ClientResponse clientResponse) {
-        return clientResponse.bodyToMono(String.class).flatMap(body -> {
-            try {
-                JsonNode node = new ObjectMapper().readTree(body).path("message");
-                if (!node.isMissingNode()) {
-                    emitStudyCreationError(studyUuid, studyName, userId, isPrivate, node.asText());
-                }
-            } catch (JsonProcessingException e) {
-                if (!body.isEmpty()) {
-                    emitStudyCreationError(studyUuid, studyName, userId, isPrivate, body);
+    private Mono<? extends Throwable> handleStudyCreationError(UUID studyUuid, String studyName, String userId, boolean isPrivate, ClientResponse clientResponse, String serverName) {
+        return clientResponse.bodyToMono(String.class)
+            .switchIfEmpty(Mono.just(serverName + ": " + clientResponse.statusCode()))
```

Comment:
This case corresponds to what type of errors ?
When we import the testCase_`import_errors_no_message_in_response_body.xiidm` file, we got no error message int the snackbar because `node.asText()` is empty !

Reply:
If an error has occurred without message in response body
The testCase_import_errors_no_message_in_response_body.xiidm file is used only in test code, to simulate that kind of error, but does not generate an error when we import it in gridsuite study-app

Reply:
But we never run this code